smoke_testing/test_scan.py

```
@@ -62,16 +62,28 @@ def scan_all_format_pdf(kubescape_exec: str):
 
 
 def scan_from_stdin(kubescape_exec: str):
-    return smoke_utils.run_command(command=["cat", single_file, "|", kubescape_exec, "--keep-local", "scan", "framework", "nsa", "-"])
+    import subprocess
+    with open(single_file, 'r') as f:
+        try:
+            output = subprocess.check_output([kubescape_exec, "scan", "framework", "nsa", "-", "--keep-local"], stdin=f, stderr=subprocess.STDOUT)
```

Ruff (0.16.1)

[error] `subprocess` call: check for execution of untrusted input

(S603)